### PR TITLE
Update abseil-cpp to LTS 20210324.1 version

### PR DESCRIPTION
**Pull request by inglor**
Fixes: #85

Signed-off-by: Leonidas Spyropoulos <artafinde@gmail.com>

### CODE REVIEW

File: subprojects/abseil-cpp.wrap [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+source_url = https://github.com/abseil/abseil-cpp/archive/20210324.1.tar.gz
+source_filename = abseil-cpp-20210324.1.tar.gz
+source_hash = 441db7c09a0565376ecacf0085b2d4c2bbedde6115d7773551bc116212c2a8d6
 
-source_url = https://github.com/abseil/abseil-cpp/archive/20200225.2.zip
-source_filename = abseil-cpp-20200225.2.zip
-source_hash = f342aac71a62861ac784cadb8127d5a42c6c61ab1cd07f00aef05f2cc4988c42
+patch_url = https://wrapdb.mesonbuild.com/v1/projects/abseil-cpp/20210324.1/1/get_zip
+patch_filename = abseil-cpp-20210324.1-1-wrap.zip
+patch_hash = 501e9656ecbeb0fbec176e1bbd94e83fab372af9e212390d7ab072ad3b6c75a8
```
Comment (polyzen) [inline subprojects/abseil-cpp.wrap:9]: I got `91ff1bca7a181cd1cdd95b2bcf2119acbab9a24ae5ef6a78185a02fa034f41da`. 🤔

Comment (Foxboron) [reply subprojects/abseil-cpp.wrap:9]: It's a zip file with embedded timestamp, probably.

Comment (inglor) [reply subprojects/abseil-cpp.wrap]: @polyzen yeah that's weird. I redownloaded and now it seems like the one you linked pushed an update

**Comment by inglor** [comment]
Superseeded from #98

**Pull request closed without merging**